modules/_labs/badge/react/eslintrc.js

```diff
@@ -0,0 +1,116 @@
+const prettierConfig = require('./.prettierrc.js');
+
+module.exports = {
```

Comment:
Pretty sure we can just `require('../../../../eslintrc.js')` and avoid all this duplication

Reply:
I wonder if that will that work if someone only pulls in this module is installed individually. 🤔

Reply:
Linting is generally only used for dev within the context of the entire monorepo. This file won't even be included in what gets published to npm

Reply:
Yep. Good call. Will update. 👍